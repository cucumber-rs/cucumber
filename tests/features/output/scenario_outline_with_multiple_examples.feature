Feature: Outline

  @original
  Scenario Outline: foo
    Given foo is <bar1>
    When foo is <bar2>
    Then foo is <bar3>

    @examples
    Examples:
      | bar1 | bar2 | bar3 |
      |  0   |  1   |  2   |

    @other-examples
    Examples:
      | bar1 | bar2 | bar3 |
      |  3   |  4   |  5   |
