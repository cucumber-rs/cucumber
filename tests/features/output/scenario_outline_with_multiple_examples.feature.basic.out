Feature: Outline
  Scenario Outline: foo
   ✔  Given foo is 0
   ✔  When foo is 1
   ✔  Then foo is 2
  Scenario Outline: foo
   ✘  Given foo is 3
      Step failed:
      Defined: tests/features/output/scenario_outline_with_multiple_examples.feature:5:5
      Matched: tests/output.rs:11:1
      Step panicked. Captured output: assertion failed: `(left == right)`
        left: `0`,
       right: `3`
